addon/mixins/oidc-authentication-route-mixin.js

```diff
@@ -21,7 +21,11 @@ export default Mixin.create(UnauthenticatedRouteMixin, {
 
   redirectUri: computed(function() {
     let { protocol, host } = location;
-    let path = this.router.urlFor(Configuration.authenticationRoute);
+    let route = this.authenticationRoute;
```

Comment:
You could simplify this by doing `this.router.urlFor(this.authenticationRoute || Configuration.authenticationRoute)`

Reply:
The Tests are still failing because of eslint, please use prettier or eslint to fix it.